devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy

```diff
@@ -0,0 +1,157 @@
+ /*
+  *  Copyright 2018 SmartThings
+  *
+  *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  *  use this file except in compliance with the License. You may obtain a copy
+  *  of the License at:
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  *  Unless required by applicable law or agreed to in writing, software
+  *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  *  License for the specific language governing permissions and limitations
+  *  under the License.
+  *  Author : Donald Kirker, Fen Mei / f.mei@samsung.com
+  *  Date : 2019-02-26
+  */
+
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition (name: "Zigbee CO Sensor", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.sensor.smoke", vid: "generic-carbon-monoxide-3") {
+		capability "Carbon Monoxide Detector"
+		capability "Sensor"
+		capability "Battery"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Health Check"
+
+		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500", outClusters: "0000", manufacturer: "ClimaxTechnology", model: "CO_00.00.00.22TC", deviceJoinName: "Ozom Smart Carbon Monoxide Sensor", mnmn: "SmartThings", vid: "generic-carbon-monoxide"
+	}
+
+	tiles {
+		multiAttributeTile(name:"carbonMonoxide", type: "lighting", width: 6, height: 4) {
+			tileAttribute ("device.carbonMonoxide", key: "PRIMARY_CONTROL") {
+				attributeState("clear", label: "clear", icon: "st.alarm.smoke.clear", backgroundColor: "#ffffff")
+				attributeState("detected", label: "MONOXIDE", icon: "st.alarm.carbon-monoxide.carbon-monoxide", backgroundColor: "#e86d13")
+			}
+		}
+
+		valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "carbonMonoxide"
+		details(["carbonMonoxide", "battery", "refresh"])
+	}
+}
+
+def installed(){
+	log.debug "installed"
+
+	if (isOzomCO()) {
+		sendEvent(name: "battery", value: 100, unit: "%", displayed: false)
+	}
+
+	refresh()
```

Comment:
needs to be `response(refresh())` I think?

Reply:
Yeah, I think installed, updated, and uninstalled need it... This was copied from the Zigbee Smoke, which leads me to believe that it was never tested...